scripts/prod_deploy/nginx.conf

```diff
@@ -0,0 +1,18 @@
+server {
+    listen 80;
+    server_name 54.174.175.215;
```

Comment:
Oh, and there is a gotcha when setting the Host header, you need this if you ever don't run on port 80 or 443.  We got bitten by this in Wakari.  I don't know why the examples fail to mention this.  Most people must always run on 80 or 443.

```
proxy_set_header Host $host:$server_port;
```
If you don't do that and say nginx is running on port 81, your app will generate urls like this:  `http://<hostname>/myapp` instead of `http://<hostname>:<port>/myapp.